Configuration/HLT/python/autoCondHLT.py

```diff
@@ -18,11 +18,11 @@
 l1Menus= {
     'Fake'         : ( ','.join( [ 'L1GtTriggerMenu_L1Menu_Collisions2012_v3_mc'             , l1MenuRecord,connectionString, l1MenuLabel, "2015-10-26 12:00:00.000"] ), ),
     'Fake1'        : ( ','.join( [ 'L1Menu_Collisions2015_25nsStage1_v5'                     , l1MenuRecord,connectionString, l1MenuLabel, "2015-10-26 12:00:00.000"] ), ),
-    'FULL'         : ( ','.join( [ 'L1Menu_Collisions2016_v6_xml'                            ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-07-19 10:00:00.000"] ), ),
-    'GRun'         : ( ','.join( [ 'L1Menu_Collisions2016_v6_xml'                            ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-07-19 10:00:00.000"] ), ),
+    'FULL'         : ( ','.join( [ 'L1Menu_Collisions2016_v6r5_ugt_1board_xml'               ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-08-23 20:00:00.000"] ), ),
+    'GRun'         : ( ','.join( [ 'L1Menu_Collisions2016_v6r5_ugt_1board_xml'               ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-08-23 20:00:00.000"] ), ),
     '25ns10e33_v2' : ( ','.join( [ 'L1Menu_Collisions2016_v4_xml'                            ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-05-24 14:55:00.000"] ), ),
     'HIon'         : ( ','.join( [ 'L1Menu_CollisionsHeavyIons2015_v5_uGT_xml'               ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-03-04 15:00:00.000"] ), ),
-    'PIon'         : ( ','.join( [ 'L1Menu_Collisions2016_v6_xml'                            ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-07-19 10:00:00.000"] ), ),
+    'PIon'         : ( ','.join( [ 'L1Menu_Collisions2016_v6r5_ugt_1board_xml'               ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-08-23 20:00:00.000"] ), ),
```

Comment:
It depends: currently the @relval still points to the previous menu.
We should probably change it to point to the new frozen menu, and then the relvals
which of course use official GTs will fail until those GTs are updated with the new L1 menu.
How do we proceed? You make new GTs and a PR whose commit we add to this one as in the past?